custom_components/hacs/translations/en.json

```diff
@@ -68,7 +68,7 @@
         },
         "removed": {
             "title": "Repository removed from HACS",
-            "description": "{name} has been removed from HACS for {reason} visit the [HACS Panel](/hacs/repository/{repositry_id}) to remove it."
+            "description": "Because {reason}, '{name}' has been removed from HACS. Please visit the [HACS Panel](/hacs/repository/{repositry_id}) to remove it locally."
```

Comment:
```suggestion
            "description": "Because {reason}, '{name}' has been removed from HACS. Please visit the [HACS Panel](/hacs/repository/{repositry_id}) to remove it."
```